core/fx/stream.go

```diff
@@ -63,6 +78,9 @@ func From(generate GenerateFunc) Stream {
 
 // Just converts the given arbitrary items to a Stream.
 func Just(items ...interface{}) Stream {
+	if len(items) == 0 {
```

Comment:
If no this code block, what's the problem?

Reply:
When the length of `items` is 0, we can simplify the operation and reuse `empty`.

Reply:
length 0 is a rare scenario, the only thing we need to do is working correctly.

Reply:
I will delete it.